src/dnc/azext_dnc/aaz/latest/dnc/controller/_wait.py

```diff
@@ -0,0 +1,204 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "dnc controller wait",
+)
+class Wait(AAZWaitCommand):
```

Comment:
What's the use of this command ?
It seems a GET (pretty similar to the show) with a different name. Does it require some particular logic on the RP?

Reply:
We don't have any logic in the RP, but this seems to be a global command that is included with AAZ command model. I don't see any way of removing it, but it exists today as well: https://learn.microsoft.com/en-us/cli/azure/dnc/controller?view=azure-cli-latest#az-dnc-controller-wait